test/Microsoft.ApplicationInsights.AspNetCore.Tests/HostingDiagnosticListenerTest.cs

```diff
@@ -1014,7 +1021,35 @@ public void RequestTelemetryIsProactivelySampledOutIfFeatureFlagIsOn(AspNetCoreM
         [InlineData(AspNetCoreMajorVersion.One)]
         [InlineData(AspNetCoreMajorVersion.Two)]
         [InlineData(AspNetCoreMajorVersion.Three)]
-        public void RequestTelemetryIsNotProactivelySampledOutIfFeatureFlasIfOff(AspNetCoreMajorVersion aspNetCoreMajorVersion)
+        public void RequestTelemetryIsProactivelySampledInIfFeatureFlagIsOnButActivityIsRecorded(AspNetCoreMajorVersion aspNetCoreMajorVersion)
+        {
+            TelemetryConfiguration config = TelemetryConfiguration.CreateDefault();
+            config.ExperimentalFeatures.Add("proactiveSampling");
+            config.SetLastObservedSamplingPercentage(SamplingTelemetryItemTypes.Request, 0);
+
+            HttpContext context = CreateContext(HttpRequestScheme, HttpRequestHost, "/Test", method: "POST");
+            var traceParent = "00-4e3083444c10254ba40513c7316332eb-e2a5f830c0ee2c46-01";
```

Comment:
perhaps a test with -00 for Recorded flag?

Reply:
this one tests -01 request with sampled out flag set to 100% of items, so this is intentionally -01